Dockerfile

```diff
@@ -46,7 +46,7 @@ COPY apps/fz_vpn/mix.exs /var/app/apps/fz_vpn/mix.exs
 COPY apps/fz_wall/mix.exs /var/app/apps/fz_wall/mix.exs
 COPY mix.exs /var/app/mix.exs
 COPY mix.lock /var/app/mix.lock
-RUN mix do deps.get --only $MIX_ENV, deps.compile, compile
+RUN mix do deps.get, deps.compile, compile
```

Comment:
I run tests in container as well, in addition to just development. This just saves us an extra `mix deps.get`

Reply:
👍